components/brave_wallet_ui/stories/wallet-extension-panels.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+
+// Components
+import { ConnectWithSite } from '../components/extension'
+import { WalletAccountType } from '../components/extension/connect-with-site-panel'
+import { StyledExtensionWrapper } from './style'
+
+export default {
+  title: 'Wallet/Extension/Panels',
+  parameters: {
+    layout: 'centered'
+  }
+}
+
+export const _ConnectWithSite = () => {
+  const [selectedAccounts, setSelectedAccounts] = React.useState<WalletAccountType[]>([
+    {
+      id: '1',
+      name: 'Account 1',
+      address: '0x7d66c9ddAED3115d93Bd1790332f3Cd06Cf52B14'
+    }
+  ])
+  const url = 'https://app.uniswap.org'
+  const accounts: WalletAccountType[] = [
+    {
+      id: '1',
+      name: 'Account 1',
+      address: '0x7d66c9ddAED3115d93Bd1790332f3Cd06Cf52B14'
+    },
+    {
+      id: '2',
+      name: 'Account 2',
+      address: '0x73A29A1da97149722eB09c526E4eAd698895bDCf'
+    },
+    {
+      id: '3',
+      name: 'Account 3',
+      address: '0x3f29A1da97149722eB09c526E4eAd698895b426'
+    }
+  ]
+  const selectAccount = (account: WalletAccountType) => {
+    const newList = [...selectedAccounts, account]
+    console.log(newList)
```

Comment:
Perhaps either remove thse before merging, or add a string parameter as a label to each. Also if you want to keep it, consider using `console.verbose`?

Reply:
Removed! :)